src/commands/extension/extension-builder.ts

```diff
@@ -64,5 +65,58 @@ export default function extensionCommandBuilder() {
         .asyncAction(ExtensionLaunchURL.setLaunchURLHandler);
 
     extension.addCommand(launch_url);
+
+    const binding = new Command('binding').description(
```

Comment:
Why it is duplicate with src/commands/binding/binding-builder.ts ?

Reply:
removed